[WIP] Chapter 3 example from Krish

adowling2 opened the pull request:
I fixed the file name. It is important the name is XX.YY-Something.ipynb.

@mukherjee07 some issues to address:
- [x] When I run this file on my local machine (anaconda Python), I get the following error message in cell 4:
- [x] Did you make this example by yourself, or did it come from somewhere else? If it came from somewhere else, you need to credit the source material.

```
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
/anaconda3/lib/python3.7/site-packages/matplotlib/projections/__init__.py in get_projection_class(projection)
     57     try:
---> 58         return projection_registry.get_projection_class(projection)
     59     except KeyError:

/anaconda3/lib/python3.7/site-packages/matplotlib/projections/__init__.py in get_projection_class(self, name)
     24         """
---> 25         return self._all_projection_types[name]
     26 

KeyError: '3d'

During handling of the above exception, another exception occurred:

ValueError                                Traceback (most recent call last)
<ipython-input-4-e0f7b44778ef> in <module>
      1 ## Sampling and visualizing the input tri-variate normal distribution dataset
      2 data = sample_trivariate_xyz()
----> 3 scatter_3d(data)
      4 # Tip: Any .csv can be used as input with pd.dataframe command

<ipython-input-3-e97b9339f82c> in scatter_3d(data, title, columns, fig, position)
     32     position = position or 111
     33 
---> 34     ax = fig.add_subplot(position, projection='3d')
     35     ax.scatter(*(
     36         data[column]

/anaconda3/lib/python3.7/site-packages/matplotlib/figure.py in add_subplot(self, *args, **kwargs)
   1347         else:
   1348             projection_class, kwargs, key = process_projection_requirements(
-> 1349                 self, *args, **kwargs)
   1350 
   1351             # try to find the axes with this key in the stack

/anaconda3/lib/python3.7/site-packages/matplotlib/projections/__init__.py in process_projection_requirements(figure, polar, projection, *args, **kwargs)
     79 
     80     if isinstance(projection, str) or projection is None:
---> 81         projection_class = get_projection_class(projection)
     82     elif hasattr(projection, '_as_mpl_axes'):
     83         projection_class, extra_kwargs = projection._as_mpl_axes()

/anaconda3/lib/python3.7/site-packages/matplotlib/projections/__init__.py in get_projection_class(projection)
     58         return projection_registry.get_projection_class(projection)
     59     except KeyError:
---> 60         raise ValueError("Unknown projection %r" % projection)
     61 
     62 

ValueError: Unknown projection '3d'
```

adowling2 committed:
Fixed mistake in the filename.

adowling2 committed:
Added `from mpl_toolkits import mplot3d` to fix the matplotlib issue.

mukherjee07 commented:
Dr. Dowling,
1) I think the error is because of the external library "copulas". I have tested it in colab and it's working fine. But in desktop I think one has to install the dependency using conda. 
2) Yes, I have taken the functions from another source but changed some of the code. I have added their website link as a reference. Thanks a lot for pointing this out.

adowling2 commented:
@mukherjee07 I am going to close your other PR. Please make the changes to this version:
https://github.com/ndcbe/cbe67701-uncertainty-quantification/blob/Chapter-3-Krish/notebooks/03.01-Contributed_Example.ipynb

Upload the changes to the same branch you were using. I made some minor edits to your version already; I fixed the file name and I added a line to ensure it runs on anaconda.

You can use pip inside anaconda. Your code worked fine on my local machine. The issue was with matplotlib.

mukherjee07 committed:
Update 03.01-Contributed_Example.ipynb
